bin/update-version.php

```diff
@@ -25,6 +25,9 @@ function replace_version( $filename, $package_json ) {
 		if ( stripos( $line, 'Stable tag: ' ) !== false ) {
 			$line = "Stable tag: {$package_json->version}\n";
 		}
+		if ( stripos( $line, '"name": "woocommerce/woocommerce-admin",' ) !== false ) {
+			$line .= "\t\"version\": \"{$package_json->version}\",\n";
+		}
```

Comment:
You'll need to delete the previous "version" line somhow. Running `npm run bump-version` a few times adds a new version line.

![Screen Shot 2020-08-04 at 11 05 24 AM](https://user-images.githubusercontent.com/1922453/89235593-925d6000-d642-11ea-9d99-d07e9754bc26.png)

Reply:
This really should only be run for releases. Development builds won't create the version property and we should only commit the changes to the Composer JSON to release branches.

I agree that more robust handling of this version should be built - are you OK with pushing that to a follow up as to not block this PR from landing?

Reply:
Yes, totally as this would only present a problem on the second release.